src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -4984,6 +4985,38 @@ private SourceLocation getSelectionAsSourceLocation(boolean fromClick)
       int column = selPos.getColumn() + 1;
       return SourceLocation.create(file, line, column, fromClick);
    }
+   
+   @Handler
+   void onQuickAddNext()
+   {
+      if (!(docDisplay_ instanceof AceEditor))
```

Comment:
But then the `Search` class requires access to an Ace editor's underlying `session`, and exposing the internal Ace `session` from `DocDisplay` doesn't feel right either, so the alternate solution would require a lot more code. :-/